create-gh-issue.py

```diff
@@ -66,8 +73,10 @@
     except:
         pass
 if issue:
-    print("Updating comment")
-    issue.create_comment(msg)
+    print(args.quiet)
+    if args.quiet == False:
+        print("Updating comment")
+        issue.create_comment(msg)
```

Comment:
do we really need an extra arg here? why not just return if issue already exist?

Reply:
Do we use this script `create-gh-issue.py` to add comments too?

Reply:
ok, looks like `run-user-pr-test` script is using it to either create issue or add a comment

Reply:
Since adding comments was the default behavior, I did not want to touch it. I see it is used in `run-user-pr-test` here:
* https://github.com/cms-sw/cms-bot/blob/master/run-user-pr-test#L37
* https://github.com/cms-sw/cms-bot/blob/master/run-user-pr-test#L88

Let me check if we really use the comment feature.